### ISSUE TITLE
Better warning when Coil:Heating:Water sizes to zero

### ISSUE THREAD

**Issue by BobHenninger** [NREL/EnergyPlus#4281]
Running test file in v8.1 gives the following warning which needs to be improved

   ** Warning ** The design coil load is zero for Coil:Heating:Water VAV SYS 1 HEATING COIL
   **   ~~~   ** The autosize value for maximum water flow rate is zero
   **   ~~~   ** To change this, input a value for UA, change the heating design day, or lower
   **   ~~~   **   the system heating design supply air temperature

At a minimum it should say "or RAISE the system heating design supply air temperature" not LOWER.
Helpdesk Ticket 5364
Example file
HVACTemplate-5ZoneVAVWaterCooled-ForceZeroCapHeatiCoil.idf

**Comment by JasonGlazer** [comment]
Verified that this is till a bug in EnergyPlus 8.4.0

@mjwitte in the original description it says "at minimum" what else should be addressed?

**Comment by JasonGlazer** [comment]
@mjwitte @Myoldmopar For the test file the preheat and the supply air temperature were the same so I changed the warning to not only say "raise" instead of lower but also to cover the preheat issues. It now will read:

```
   ** Warning ** The design coil load is zero for Coil:Heating:Water VAV SYS 1 HEATING COIL
   **   ~~~   ** The autosize value for maximum water flow rate is zero
   **   ~~~   ** To change this, input a value for UA, change the heating design day, or raise the
   **   ~~~   **   system heating design supply air temperature. Also check to make sure the Preheat
   **   ~~~   **   Design Temperature is not the same as the Central Heating Design Supply Air Temperature. 
```

Does that seem like a good error message?

**Comment by mjwitte** [comment]
@JasonGlazer That looks good.  The preheat part should only apply if there actually is a preheat coil, but this is ok for here.

**Comment by JasonGlazer** [comment]
@mjwitte Would it make more sense to make a new warning message when the preheat and supply temperatures are specified as the same in Sizing:System and remove that from this one?

**Comment by mjwitte** [comment]
Perhaps, but it should only fire if both coils are present and I'm not sure where in the process that is known.

**Comment by JasonGlazer** [comment]
I am not sure either I think that would be tricky to try to figure out. I guess I will just go with the warning I showed a few messages ago.

**Comment by Myoldmopar** [comment]
Closed via #5342
